clang_delta/RemoveUnusedStructField.cpp

```diff
@@ -280,6 +280,7 @@ void RemoveUnusedStructField::getInitExprs(const Type *Ty,
 
     if (FD == TheFieldDecl) {
       InitExprs.push_back(Init);
+      //TODO: Is the (void)VecSz necessary?
       TransAssert((VecSz == 1) && "Bad IndexVector size!"); (void)VecSz;
```

Comment:
Ah ok, I was just wondering if it there might be some side-effects which I have overlooked.